crates/holochain/src/conductor/space.rs

```diff
@@ -156,6 +158,69 @@ impl Spaces {
         })
     }
 
+    /// Get the holochain conductor state
+    pub async fn get_state(&self) -> ConductorResult<ConductorState> {
```

Comment:
`Spaces` is becoming an amorphous blob of many unrelated things. I don't say that because of this PR, but the fact that this had to be moved here seems to illustrate that point. We can probably reorganize things a bit better some day.

Reply:
:100: